From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Tue, 30 Aug 2011 22:16:13 +0200
Subject: Use C locale when calling date.

Last-Update: 2011-08-16
Forwarded: no
---
 example.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/example.py b/example.py
index ffa78e7..59ca99c 100644
--- a/example.py
+++ b/example.py
@@ -5,7 +5,7 @@ from dateutil.parser import *
 from datetime import *
 import subprocess
 import os
-now = parse(subprocess.getoutput("date"))
+now = parse(subprocess.getoutput("LC_ALL=C date"))
 today = now.date()
 year = rrule(YEARLY, bymonth=8, bymonthday=13, byweekday=FR)[0].year
 rdelta = relativedelta(easter(year), today)
